### PR TITLE
Convert2RHEL component change to Registration

### PR DESCRIPTION
**Pull request by shweta83**

### CODE REVIEW

File: tests/foreman/api/test_convert2rhel.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 :CaseLevel: Integration
 
-:CaseComponent: Convert2rhel
+:CaseComponent: Registration
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_convert2rhel.py:9]: Registration seems irrelevant here, Can you add some description about this change?

**Review by lhellebr — CHANGES_REQUESTED** [review]
This file has two tests, both of them test specifically convert2rhel. Since we have a component for that, I don't see any reason not to use it.

Are there plans to drop that component? If yes, may I suggest using a different component, like RemoteExecution or Ansible?

Registration really seems out of place - while registration happens as a part of the test, it's not what is being tested.

**Comment by Gauravtalreja1** [comment]
@shweta83 can this be closed in favour of PR #10863 ?

**Comment by shweta83** [comment]
> @shweta83 can this be closed in favour of PR #10863 ?

Yes we can close this one.

**Comment by shweta83** [comment]
Closing

**Pull request closed without merging**